.travis.yml

```diff
@@ -16,7 +17,15 @@ jobs:
   - stage: Integration tests
     script:
     - sudo /etc/init.d/mysql stop
+    - sudo sysctl fs.inotify.max_user_watches=524288; sudo sysctl -p
     - git clone https://github.com/streamr-dev/streamr-docker-dev.git
+    - sudo ifconfig docker0 10.200.10.1/24
+    - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start 4"
```

Comment:
I think there's redundancy here: `streamr-docker-dev start engine-and-editor` will actually start `engine-and-editor` and all its dependencies, including all the services in "4", so I guess this line could just be removed?